lib/Bio/EnsEMBL/DataCheck/Checks/CheckDuplicatedTaxaNames.pm

```diff
@@ -0,0 +1,57 @@
+=head1 LICENSE
+
+Copyright [2018-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CheckDuplicatedTaxaNames;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME           => 'CheckDuplicatedTaxaNames',
+  DESCRIPTION    => 'Check that the ncbi_taxa_name contains only unique rows',
+  GROUPS         => ['compara'],
```

Comment:
Not at the moment, I think the groups we have currently are: `compara`, `compara_pairwise_alignments`, `compara_multiple_alignments`, `compara_syntenies`, `compara_protein_trees`. If we were to add a new group, `compara_shared`, since these DCs are split between multiple PRs, some accepted and some waiting for approval, and with no rush for release/100 now, it might be worth waiting until all PRs are accepted and adding the group in one go to all the affected DCs?

Reply:
I think what was in `ComparaShared` would probably go in `compara`. The convention is that the group with that name has fundamental checks that apply to all databases of that type, including master.

Reply:
Another difference is that in Java we could compose groups. ComparaMaster was defined as ComparaShared + ComparaMasterOnly (although the latter was empty).
I agree with @carlacummins : ww'll need `compara_master` too to test the master database. I don't mind waiting for all the DCs to be merged and then we can review the groups and introduce `compara_master`